Darkly/LDClientManager.m

```diff
@@ -26,6 +27,7 @@ +(LDClientManager *)sharedInstance {
         
         [[NSNotificationCenter defaultCenter] addObserver:sharedApiManager selector:@selector(willEnterForeground) name:UIApplicationWillEnterForegroundNotification object:nil];
         [[NSNotificationCenter defaultCenter] addObserver:sharedApiManager selector:@selector(willEnterBackground) name:UIApplicationWillResignActiveNotification object:nil];
+        [[NSNotificationCenter defaultCenter] addObserver:sharedApiManager selector:@selector(syncWithServerForEvents) name:kLDBackgroundFetchInitiated object:nil];
```

Comment:
I think this is the most minimalistic approach we can provide, and given a proper documentation, it will be quite easy for the developers to integrate.